### ISSUE TITLE
Add the autokey-shell man page.

### ISSUE THREAD

**Issue by Elliria** [autokey/autokey#870]
### AutoKey is a Xorg application and will not function in a Wayland session. Do you use Xorg (X11) or Wayland?

Xorg

### Has this issue already been reported?

- [X] I have searched through the existing issues.

### Is this a question rather than an issue?

- [X] This is not a question.

### What type of issue is this?

Documentation

### Choose one or more terms that describe this issue:

- [ ] autokey triggers
- [ ] autokey-gtk
- [ ] autokey-qt
- [ ] beta
- [ ] bug
- [ ] critical
- [X] development
- [X] documentation
- [ ] enhancement
- [ ] installation/configuration
- [ ] phrase expansion
- [ ] scripting
- [X] technical debt
- [ ] user interface

### Which AutoKey GUI did you use?

None

### Can you briefly describe the issue?

The autokey-shell man page is missing from the project and needs to be added.

### Can the issue be reproduced?

Always

### What are the steps to reproduce the issue?

Type `man autokey-shell` on the command-line.

### What should have happened?

The man page should open.

### What actually happened?

A message is displayed letting you know there is no such man page.

### Anything else?

The [Running AutoKey scripts interactively on a shell](https://github.com/autokey/autokey/blob/4b87b32452cd41d2b98341db220e99cf4f9e34d6/new_features.rst#running-autokey-scripts-interactively-on-a-shell) section in the **new_features.rst** page gives a basic description of **autokey-shell**, which will help in creating the page.

Also, this search for [autokey-run](https://github.com/autokey/autokey/search?q=autokey-run&type=code) and this search for [autokey-shell](https://github.com/search?q=repo%3Aautokey%2Fautokey+autokey-shell&type=code) in AutoKey's code will prove useful in figuring out all the places that changes will need to be made to add a man page.

Last, but not least, when this is done, add links to this man page in the [autokey-gtk](https://github.com/autokey/autokey/blob/develop/doc/man/autokey-gtk.1) and [autokey-qt](https://github.com/autokey/autokey/blob/develop/doc/man/autokey-qt.1) and [autokey-run](https://github.com/autokey/autokey/blob/develop/doc/man/autokey-run.1) man pages.

**Comment by Elliria** [comment]
This can be assigned to me, @josephj11, although I'll probably need help with making sure I've got it added correctly in all the right places when the time comes.

**Comment by josephj11** [comment]
The help comes from reqesting reviews ... Being assigned doesn't imply that you're on your own to get it done.

**Comment by Elliria** [comment]
Yep. It was a heads-up that I'll likely need help, but you can assign it to me because I intend to get it done.
